recipes/libcurl/all/conanfile.py

```diff
@@ -195,15 +186,19 @@ def requirements(self):
         if self.options.with_c_ares:
             self.requires("c-ares/1.18.1")
 
+    def package_id(self):
+        del self.info.settings.compiler
```

Comment:
what is that?

Reply:
I didn't manage to find why I pushed this commit.